pkg/cidata/cidata.TEMPLATE.d/boot/00-alpine-user-group.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# Remove the user embedded in the image,
+# and use cloud-init for users and groups.
+test -f /etc/alpine-release || exit 0
+
```

Comment:
Although unlikely, should this have a safeguard for the case where the host user is also called `alpine`? Otherwise we would delete the actual user again on each reboot:

```bash
test [ "$LIMA_CIDATA_USER" != "alpine" ] || exit 0
```

Reply:
Right, if your parents called you "alpine" as a kid you would end up with a boot loop